tesseract_visualization/include/tesseract_visualization/markers/toolpath_marker.h

```diff
@@ -0,0 +1,25 @@
+#ifndef TESSERACT_VISUALIZATION_MARKERS_TOOLPATH_MARKER_H
+#define TESSERACT_VISUALIZATION_MARKERS_TOOLPATH_MARKER_H
+
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <vector>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include <tesseract_visualization/markers/marker.h>
+
+namespace tesseract_visualization
+{
+/** @brief An arrow defined by two points */
+class ToolpathMarker : public Marker
+{
+public:
+  ToolpathMarker() { type_ = static_cast<int>(MarkerType::TOOLPATH); }
+
+  bool show_path{ true };
+  bool show_axis{ true };
+  std::vector<std::vector<Eigen::Isometry3d>> toolpath;
```

Comment:
Need special Eigen allocator for STL vectors

Reply:
The Isometry3d is not a [fixed size vectorizable type](https://eigen.tuxfamily.org/dox/group__TopicFixedSizeVectorizable.html). Have you ran into issues?

Reply:
Are you sure? I'm not sure that list is comprehensive, and it includes both the `Affine3d` and `Matrix4d` which are the same size matrix as `Isometry3d`

Reply:
Your right, I believe the list is not up to date. Looking at the code is is still a 4x4 matrix. Also I looked at tesseract_common types and I am actually aligning it so I will use that type. Good catch.